code/datums/diseases/advance/advance.dm

```diff
@@ -67,40 +76,70 @@
 		),
 		list( //level 5
 			/datum/reagent/drug/space_drugs,
-			/datum/reagent/medicine/mannitol,
-			/datum/reagent/medicine/synaptizine,
 			/datum/reagent/cryptobiolin,
+			/datum/reagent/medicine/coagulant,
+			/datum/reagent/medicine/ammoniated_mercury,
+			/datum/reagent/medicine/neurine,
+			/datum/reagent/toxin/heparin,
 		),
 		list( // level 6
 			/datum/reagent/medicine/antihol,
 			/datum/reagent/medicine/inacusiate,
 			/datum/reagent/medicine/oculine,
-			/datum/reagent/phenol,
+			/datum/reagent/kronkus_extract,
```

Comment:
It is not reasonably possible kronkus seeds in most rounds as it is rare maint loot. It is very likely that simply having this virus cure makes it almost impossible to cure via any typical means beyond getting lucky via sleeping. 

There is exactly one way to force it into the round, and it involves an equally large amount of RNG by buying a shitton of garbage crates out of cargo, and it is still a very slim possibility you get this at all.

Reply:
Valid.